grafana/resource_machine_learning_holiday.go

```diff
@@ -0,0 +1,202 @@
+package grafana
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/grafana/machine-learning-go-client/mlapi"
+)
+
+func ResourceMachineLearningHoliday() *schema.Resource {
+	return &schema.Resource{
+
+		Description: `
+A holiday describes time periods where a time series is expected to behave differently to normal.
+
+To use a holiday in a job, use its id in the "holidays" attribute of a "grafana_machine_learning_job":
+
+		resource "grafana_machine_learning_job" "test_job" {
+			...
+			holidays = [
+				grafana_machine_learning_holiday.my_holiday.id
+			]
+		}
+`,
```

Comment:
This will probably look weird in the docs without code blocks

```suggestion
		Description: `
A holiday describes time periods where a time series is expected to behave differently to normal.

To use a holiday in a job, use its id in the "holidays" attribute of a "grafana_machine_learning_job":

` + 
"```terraform`" +
`
resource "grafana_machine_learning_job" "test_job" {
    ...
    holidays = [
        grafana_machine_learning_holiday.my_holiday.id
    ]
}
` + "```",
```

Reply:
Good spot! I've pushed a manual fix as there was an extra ` in the suggestion, thanks though 🙂